data/incipias/first contact.txt

```diff
@@ -0,0 +1,287 @@
+			label haihelp
+			`    Now the Hai turn to you with a very serious expression, "We have promised you to help you and now we will do so. Please come back in no earlier than four days to take a delegation of us to the Quarg ringworld. We need some time to prepare some stuff. You will meet us on Mirrorlake." You agree and say goodbye.`
+		event "Hai preparation time" 4
+
+event "Hai preparation time"
+
+mission "wanderer peace prove"
+	landing
+	name "Prove of Peace"
+	source
+		government "Wanderer"
+		not planet "Vara K'chrai"
+	destination "Vara K'chrai"
+	to offer
+		and
+			has "searching for a prove"
+			and
+				not "coalition peace prove: offered"
+				"reputation: Wanderer" >= 5
+	on offer
+		conversation
+			`Now that you are on a wanderer world you decide this would be a good time for asking them about the prove the Hai requested from you. You head for a near embassy.`
+			`    At the embassy an older Wanderer ask you what you are there for. You remember the Hai telling you that you should not tell the other species about the Incipias, so you search for something else to tell them.`
+			choice
+				"I am searching for something unique that presents your species best."
+					goto unique
+				"I am gathering proves of friendship from all over the galaxy."
+			`   "That sounds interesting, let me think about it." The Wanderer becomes silent for some time. Then he says, "I got an idea, but I need to ask some people about it first. Please wait here for a second." He leaves the room without waiting for your respond. Forced to wait for him, you ask yourself what this gift of friendship of the wanderers could be.`
+				goto c1
+			label unique
+			`    "Why are you searching for something like this? That is an quite unusual request."`
+			choice
+				"I am a passionate collector."
+				"I want something to remember you."
+			`    "Let me think about this. What would present our species best..." The Wanderer gets silent while thinking. Than he suddenly stands up, "If I get the permission to give you this, you will get a perfect item that presents our species best. Let me go check it." The Wanderer leaves the room.`
+			label c1
+			`    Time is passing and you get a little bit nervous, but after some time the wanderer finally returns. He immedeatly begins to speak. "I just got the permission", he says "You will have to travel to <destination> to get this. You will receive it at the embassy there, I already contacted some people there." You ask him what it his but all he tells you is "It is a surprise, you will see when you get there. Take this." He hands you a small card. "This will tell the people there that you are the right one." He says goodbye to you and you leave the embassy.`
```

Comment:
```suggestion
			`    Time passes and you get a little bit nervous, but after some time the Wanderer finally returns. He immediately begins to speak. "I just got permission", he says "You will have to travel to <destination> to get this. You will receive it at the embassy there, I already contacted some people there." You ask him what it his but all he tells you is "It is a surprise, you will see when you get there. Take this." He hands you a small card. "This will tell the people there that you are the right one." He says goodbye to you and you leave the embassy.`
```